node/node_test.go

```diff
@@ -1324,6 +1338,9 @@ func TestNode_StartHeartbeatShouldWorkAndCallSendHeartbeat(t *testing.T) {
 				return mock.NewStorerMock()
 			},
 		}),
+		node.WithNodesCoordinator(&mock.NodesCoordinatorMock{}),
+		node.WithEpochStartTrigger(&mock.EpochStartTriggerStub{}),
+		node.WithEpochStartSubscriber(&mock.EpochStartNotifierStub{}),
```

Comment:
please rename the option, as this is not a subscriber but a notifier.

Reply:
done